aws/resource_aws_lightsail_domain_test.go

```diff
@@ -134,5 +134,5 @@ provider "aws" {
 resource "aws_lightsail_domain" "domain_test" {
   domain_name = "%s"
 }
-`, lightsailDomainName)
+`, lightsailDomainName) //lintignore:AT004
```

Comment:
Per https://github.com/hashicorp/terraform-provider-aws/pull/9080 description and recent similar changes for other region-specific testing (e.g. https://github.com/hashicorp/terraform-provider-aws/pull/16140), will refactor this post-merge and verify in AWS Commercial and AWS GovCloud (US). If you're curious about this esoteric subject (why I'm not requesting you to make these changes 😉 ), please see this very recent [addition in the Contributing Guide](https://github.com/hashicorp/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md#service-specific-region-acceptance-testing).